aten/src/ATen/Context.cpp

```diff
@@ -494,6 +494,10 @@ at::BlasBackend Context::blasPreferredBackend() {
   // hipblaslt support for all archs is not as complete as hipblas
   if (blas_preferred_backend == at::BlasBackend::Cublaslt) {
     static const bool hipblaslt_unsupported = []() {
+      if(!hasCuBLASLt())
+      {
+          return false;
```

Comment:
Shouldn't we return `true` here?  If blaslt is not detected ( `!hasCuBLASLt` ) then certainly `hipblaslt_unsupported = true`?
```suggestion
          return true;
```